### ISSUE TITLE
Large year value causes `Cannot read property 'length' of undefined`

### ISSUE THREAD

**Issue by zarjay** [gpbl/react-day-picker#717]
Entering a large year value into `DayPickerInput` causes this error to  be thrown:

```
TypeError: Cannot read property 'length' of undefined
    at Object.f [as getWeekArray] (Helpers.js:112)
```

To reproduce, enter in a date with large year value into a DayPickerInput field (e.g., 2000008-08-08). (The issue doesn't occur for large month or day values.)

**Comment by gpbl** [comment]
Thanks for finding this out, it seems we need to check the year here:
https://github.com/gpbl/react-day-picker/blob/master/src/DayPickerInput.js#L74
otherwise we go beyond the supported dates.

**Comment by gpbl** [comment]
Thanks this has been fixed in v7.1.9
